read211_tests.py

```diff
@@ -0,0 +1,92 @@
+# BuildingSync(R), Copyright (c) 2015-2018, Alliance for Sustainable Energy, LLC.
```

Comment:
@jasondegraw can you update these to 2019. I'm going to merge down for now so that we I can get the other PR's reviewed.

Reply:
@nllong Yes, I'll make the change on the last PR.